src/main/java/org/trigon/commands/GearRatioCalculationCommand.java

```diff
@@ -23,14 +23,27 @@ public class GearRatioCalculationCommand extends Command {
     private final DoubleConsumer runGearRatioCalculation;
     private final String subsystemName;
     private final double backlashAccountabilityTimeSeconds;
-    private final LoggedDashboardNumber movementVoltage;
+    private final LoggedNetworkNumber movementVoltage;
 
     private double startingRotorPosition;
     private double startingEncoderPosition;
     private double gearRatio;
     private double startTime;
     private boolean hasSetStartingPositions = false;
 
+    /**
+     * Creates a new GearRatioCalculationCommand.
+     * This constructor takes a motor to run the gear ratio calculation on, and to measure the distance the rotor travels.
+     * It also takes an encoder to measure the distance traveled.
+     *
```

Comment:
Talk about the backlash thing

Reply:
This constructor doesn't have the backlash accountabliity. So no.

Reply:
Talk about what you do (or don't do) with it